build/hero_abilities.json

```diff
@@ -0,0 +1,4855 @@
+{
+  "npc_dota_hero_antimage": {
+    "abilities": [
+      "antimage_mana_break",
+      "antimage_blink",
+      "antimage_spell_shield",
+      "antimage_mana_void"
+    ],
+    "talents": [
+      {
```

Comment:
If it's implicit that the talents are ordered 1/1/2/2/3/3/4/4 maybe we can just put the string and save some space?